nipype/interfaces/afni/utils.py

```diff
@@ -1516,6 +1516,92 @@ def _list_outputs(self):
         return outputs
 
 
+class NwarpAdjustInputSpec(CommandLineInputSpec):
+    warps = InputMultiPath(
+        File(exists=True),
+        minlen=5,
+        mandatory=True,
+        argstr='-nwarp %s',
+        desc='List of input 3D warp datasets')
+    in_files = InputMultiPath(
+        File(exists=True),
+        minlen=5,
+        mandatory=False,
+        argstr='-source %s',
+        desc='List of input 3D datasets to be warped by the adjusted warp '
+             'datasets.  There must be exactly as many of these datasets as '
+             'there are input warps.')
+    out_file = File(
+        desc='Output mean dataset, only needed if in_files are also given. '
+             'The output dataset will be on the common grid shared by the '
+             'source datasets.',
+        argstr='-prefix %s',
+        mandatory=False,
+        name_source='in_files',
+        name_template='%s_NwarpAdjust',
+        keep_extension=True,
+        xand=['in_files'])
```

Comment:
you can remove this since `in_files` is mandatory. also there is no `xand` in the current spec. so `make specs` would have returned a violation.

Reply:
actually I am trying to make `in_files` not mandatory, because what is needed here is only the warps

Reply:
I believe what you want is `requires=['in_files']`. See https://nipype.readthedocs.io/en/latest/devel/interface_specs.html.